pwndbg/commands/ptmalloc2.py

```diff
@@ -144,6 +144,12 @@ def print_no_arena_found_error(tid=None) -> None:
             f"No arena found for thread {message.hint(tid)} (the thread hasn't performed any allocations)."
         )
     )
+    print(
+        message.hint(
+            "Use `set libthread-db-search-path <path>` if the path contains a compatible libthread_db.\n"
+            "Then rerun the program."
```

Comment:
We can leave this as-is, but I wonder: shall we inform users what libthread-db is? I only discovered it today myself.

Shall we maybe also link to https://sourceware.org/gdb/current/onlinedocs/gdb.html/Threads.html#:~:text=set%20libthread%2Ddb%2Dsearch%2Dpath%20%5Bpath%5D ? Is this any helpful? Ugh!

Reply:
At that moment the `heap` commands and friends doesn't work. Users could google the configuration for more info. I was trying to be concise in the printing message.